masterlist.yaml

```
@@ -9884,6 +9884,31 @@ plugins:
         util: '[TES4Edit v4.0.3j (RC2 for 4.0.4)](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 9
 
+  - name: 'sunk ships.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/13467/'
+        name: 'Ship Wrecks'
+    msg:
+      - <<: *wildEdits
+        subs: [ 'Remove **00033569, 000CA158, 00015C3D, 00006622 & 01006EC7** from **sunk ships.esp**.' ]
```

Since the index of the mod isn't always going to be 01 & deleting the unused cell doesn't really benefit the user, it's best to leave it out. Also, I slightly adjusted the message formatting to make it consistent with other wild edit messages.

```suggestion
        subs: [ 'Remove **00033569**, **000CA158**, **00015C3D**, & **00006622** from **sunk ships.esp**.' ]
```